source/pages/basics/downloads.rst

```diff
@@ -1,76 +1,106 @@
+    :local:
 
 .. _downloads-landing:
 
-==========================
 Download files directly
-==========================
+=======================
+
+To download files directly using the browser you need to navigate to the landing page of the deposit that contains the desired files. Unregistered users can only download files of objects that are publicly accessible and staged. Files of objects that are privately shared can only be downloaded by users that have explicit access to these objects. If an object's access level is set to registered, only registered and logged in users can access the files.
 
-To download files directly from the Data Repository you need to navigate to the landing page of the deposit you want to download the files from. Unregistered users can only download files of objects that are publically accessible. Files of objects that are privately shared can only be downloaded by users that have explicit access to these objects. If an object's access level is set to registered, only registered and logged in users can access the files.
+You can download the entire deposit using the ``Download as`` button in the top left corner of the deposit page. To download a single file click the ``Download`` button next to the file. A file can be *online* or *offline*. If a file is *online* it can be downloaded immediately. If a file is *offline* it needs to be staged before it is available for download. To stage a file, login to the data repository and click the ``Request`` button.
 
-You can download single files by clicking on the **download** link next to the file name. If the status of the file is **offline**, it means it cannot be downloaded directly, but first needs to be staged. A request to stage the file can be made by clicking the **request** link.
+To download multiple files from the same deposit, select multiple files and use the ``Download as zip`` button to download the selection. If you want to bulk download multiple offline files, stage the files first or download the files using a script. See the next section for more information.
 
 .. image:: ../img/deposit-landing-page-file-table.png
    :align: center
    :width: 75%
 
-.. _downloads-basket:
+.. _download-script:
+
+Download using a script
+=======================
 
-==========================
-Adding objects or files to your basket
-==========================
+The recommended method to download multiple deposits or entire collections is the download script. The download script is a tool that automatically downloads all objects and files in your basket or favourites. To download objects using the download script is a three step process:
 
-To download several files at the same time, your can add objects or files to your basket. This can be done by clicking on the 'Add to basket' button in the top-right corner, or by selecting individual files in the files table and then clicking on the basket icon button directly above the table.
+1. install and configure the script;
```

Comment:
This is good. Let's add another step called 'Add an API token' (as is described later in the section).